prymer/primer3/primer3.py

```diff
@@ -228,13 +227,23 @@ def __init__(
         self,
         genome_fasta: Path,
         executable: Optional[str] = None,
-        variant_lookup: Optional[VariantLookup] = None,
+        vcfs_or_lookup: Optional[list[Path] | VariantLookup] = None,
```

Comment:
**question** Would it be simpler or more complicated if there were two separate arguments, `variant_vcfs` and `variant_lookup`, which were mutually exclusive?

Reply:
Since they are mutually exclusive, I kinda like doing it this way.  Though I wonder if the type would be clearer written as `list[Path] | VariantLookup | None`?

Reply:
I might also suggest calling this parameter `variants` or something else that can work for either the lookup or the vcfs.